modules/ocf_decal/manifests/init.pp

```diff
@@ -14,10 +14,24 @@
     system  => true,
   }
 
-  file { '/opt/ocfdecal':
-    ensure  => directory,
-    owner   => ocfdecal,
-    group   => ocfdecal,
-    require => User['ocfdecal'];
+  file {
+    '/opt/ocfdecal':
+      ensure  => directory,
+      owner   => ocfdecal,
+      group   => ocfdecal,
+      require => User['ocfdecal'];
+    '/etc/decal_mysql.conf':
+      source  => 'puppet:///private/mysql.conf',
+      owner   => ocfdecal,
+      group   => ocfstaff,
+      mode    => '0660',
+      require => User['ocfdecal'];
+  }
+
+  vcsrepo { '/opt/share/decal-utils':
```

Comment:
Looks good, not sure how we're going to handle the terraform stuff though, but that can happen later I suppose.

Reply:
terraform @ decal is done I think, if we can't fix the broken tfstate thing